lib/iris/tests/unit/fileformats/pp_rules/test_convert.py

```diff
@@ -91,19 +91,34 @@ def _is_sigma_coord(coord):
                 coord.units.is_dimensionless())
 
     @staticmethod
-    def _is_soil_model_level_number_coord(coord):
+    def _is_deep_soil_model_level_number_coord(coord):
         return (coord.long_name == 'soil_model_level_number' and
                 coord.units.is_dimensionless() and
                 coord.attributes['positive'] == 'down')
 
-    def test_soil_levels(self):
```

Comment:
Apologies, rename confusion ...

`test_soil_levels` is still done but is called `test_deep_soil_levels`, and there is also an associated `test_deep_soil_depth`